server/chat-plugins/room-events.ts

```diff
@@ -47,7 +47,7 @@ export const commands: ChatCommands = {
 		edit: 'add',
 		add(target, room, user) {
 			if (!room.chatRoomData) return this.errorReply("This command is unavailable in temporary rooms.");
-			if (!this.can('ban', null, room)) return false;
+			if (!user.authAtLeast(room.eventsRank, room)) return false;
```

Comment:
this probably shouldn't fail silently.

Reply:
I have no idea why this is failing silently. It executes a proper fail when i give it something like 
> if (!user.authAtLeast(room.eventsRank)) {
				return this.errorReply(`Must be rank ${room.eventsRank} or higher to use this command.`);
			}

Reply:
Yes, that's what I'm saying it should be. It's failing silently because `authAtLeast` doesn't send any kind of message on failure, unlike `can`